prow/config.yaml

```diff
@@ -179,6 +179,143 @@ presubmits:
     rerun_command: "@k8s-bot non-cri node e2e test this"
     trigger: "@k8s-bot (non-cri node e2e )?test this"
 
+  kubernetes-security/kubernetes:
```

Comment:
Feel free to add a `TODO(fejta, spxr)` or whatever suggesting that we make this less gross in the near future :smile_cat: